lib/cosmos/tools/tlm_grapher/tabbed_plots_tool/tabbed_plots_realtime_thread.rb

```diff
@@ -16,11 +16,10 @@ module Cosmos
 
   # Thread used to gather telemetry in realtime and process it using a TabbedPlotsDefinition
   class TabbedPlotsRealtimeThread < InterfaceThread
-
     # Create a new TabbedPlotsRealtimeThread
     def initialize(tabbed_plots_config, connection_success_callback = nil, connection_failed_callback = nil, connection_lost_callback = nil, fatal_exception_callback = nil, replay_mode = false)
       if replay_mode
-        interface = TcpipClientInterface.new(System.connect_hosts['REPLAY_PREIDENTIFIED'], nil, System.ports['REPLAY_PREIDENTIFIED'], nil, tabbed_plots_config.cts_timeout, 'PREIDENTIFIED')
+        interface = TcpipClientInterface.new(System.connect_hosts['REPLAY_PREIDENTIFIED'], nil, System.ports['REPLAY_PREIDENTIFIED'], nil, nil, 'PREIDENTIFIED')
```

Comment:
I don't think the Grapher should timeout when when connecting in Replay mode. Using Replay implies a lot of time just sitting there not receiving packets.